source/linux/Foundational_Components/Kernel/Kernel_Drivers/Display/DSS7.rst

```diff
@@ -490,8 +490,8 @@ tidss supports configuration via DRM properties. These are standard DRM properti
 | GAMMA_LUT_SIZE     | crtc     | Number of elements in gammma lookup table.                                                           |
 +--------------------+----------+------------------------------------------------------------------------------------------------------+
 
-Testing tidss properties with modetest
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Testing tidss properties with modetest and kmstest
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Informal ref on line 390 of this file is now invalid. Please switch to proper ref syntax to prevent this in the future.

Reply:
Added proper ref syntax.